src/uct/api/uct_def.h

```diff
@@ -405,4 +406,62 @@ typedef ucs_status_t (*uct_tag_unexp_rndv_cb_t)(void *arg, unsigned flags,
                                                 const void *rkey_buf);
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Tuning parameters for an UCT endpoint created by @ref uct_ep_create.
```

Comment:
Parameters for creating a UCT endpoint by @ref uct_ep_create
see also https://ontariotraining.net/grammar-tip-a-or-an-with-acronyms/